src/components/feedback/Dialog.tsx

```diff
@@ -236,8 +262,9 @@ export default function Dialog({
             >
               {children}
             </Panel>
+          ) : (
+            <>{children}</>
```

Comment:
An alternative approach would be to make consumers responsible for composing the dialog and panel. It looks to me like `Dialog` is mostly just forwarding props here, plus setting a few additions and defaults (`fullWidthHeader`). This would be a more disruptive change to downstream consumers though.